controllers/common/helper.go

```diff
@@ -601,24 +601,24 @@ func IsDatabaseClusterRestoreRunning(
 	return false, nil
 }
 
-// GetBackupStorageIndexInPGBackrestRepo returns the index of the backup storage in the pgbackrest repo list.
-func GetBackupStorageIndexInPGBackrestRepo(
+// GetBackupStorageNameInPGBackrestRepo returns the name of the backup storage in the pgbackrest repo list.
+func GetBackupStorageNameInPGBackrestRepo(
```

Comment:
Just a nitpick but we are actually returning the name of the repo not the name of the backup storage, right? If that's the case we should consider renaming the function to avoid confusion with the name of the backup storage CR.

Reply:
Right, thanks for noticing this. Renamed to `GetRepoNameByBackupStorage`